server/controllers/request.js

```diff
@@ -0,0 +1,80 @@
+const Request = require("../models/Request");
+const User = require("../models/User");
+const asyncHandler = require("express-async-handler");
+
+// @route GET /request
+// @desc get requests related to a logged user
+// @access Private
+exports.getRequests = asyncHandler( async (req, res, next) => {
+  const loggedUserId = req.user.id;
+  if (!loggedUserId) {
+    res.status(401);
```

Comment:
We have middleware available that does this check for us already, I would suggest taking this logic out

Reply:
FIXED. Should I add the "protect" middleware to Request controllers?